npm install error

Using node v10.12.0 and npm v6.4.1 I get the following error when running `npm install` inside this project:

```
npm WARN prepublish-on-install As of npm@5, `prepublish` scripts are deprecated.
npm WARN prepublish-on-install Use `prepare` for build steps and `prepublishOnly` for upload-only.
npm WARN prepublish-on-install See the deprecation note in `npm help scripts` for more information.

> express-mysql-session@2.1.1 prepublish D:\VOLATILE\workspace-2\express-mysql-session-2
> npm run test # runs before `npm publish`


> express-mysql-session@2.1.1 pretest D:\VOLATILE\workspace-2\express-mysql-session-2
> npm run lint # runs before `npm test`


> express-mysql-session@2.1.1 lint D:\VOLATILE\workspace-2\express-mysql-session-2
> eslint --config .eslintrc.js *.js test/**/*.js "#" "runs" "before" "`npm" "test`"


Oops! Something went wrong! :(

ESLint: 5.15.3.
No files matching the pattern "#" were found.
Please check for typing mistakes in the pattern.

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! express-mysql-session@2.1.1 lint: `eslint --config .eslintrc.js *.js test/**/*.js "#" "runs" "before" "`npm" "test`"`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the express-mysql-session@2.1.1 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! express-mysql-session@2.1.1 pretest: `npm run lint # runs before `npm test``
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the express-mysql-session@2.1.1 pretest script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! express-mysql-session@2.1.1 prepublish: `npm run test # runs before `npm publish``
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the express-mysql-session@2.1.1 prepublish script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

Huh that's weird.

To further increase confusion, I'm now getting a different error:
Huh that's weird.

To further increase confusion, I'm now getting a different error:
```
npm WARN prepublish-on-install As of npm@5, `prepublish` scripts are deprecated.
npm WARN prepublish-on-install Use `prepare` for build steps and `prepublishOnly` for upload-only.
npm WARN prepublish-on-install See the deprecation note in `npm help scripts` for more information.

> express-mysql-session@2.1.1 prepublish *snip*\express-mysql-session
> npm run test


> express-mysql-session@2.1.1 pretest *snip*\express-mysql-session
> npm run lint


> express-mysql-session@2.1.1 lint *snip*\express-mysql-session
> eslint --config .eslintrc.js *.js test/**/*.js


> express-mysql-session@2.1.1 test *snip*\express-mysql-session
> npm run test:unit && npm run test:integration


> express-mysql-session@2.1.1 test:unit *snip*\express-mysql-session
> mocha test/unit/ --recursive --reporter spec --ui bdd

*snip*\express-mysql-session\node_modules\yargs\yargs.js:1148
      else throw err
           ^

Error: Cannot find module 'express-session'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
    at Function.Module._load (internal/modules/cjs/loader.js:506:25)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (*snip*\express-mysql-session\test\manager.js:5:15)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (*snip*\express-mysql-session\test\unit\all.js:6:15)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at *snip*\express-mysql-session\node_modules\mocha\lib\mocha.js:327:36
    at Array.forEach (<anonymous>)
    at Mocha.loadFiles (*snip*\express-mysql-session\node_modules\mocha\lib\mocha.js:324:14)
    at Mocha.run (*snip*\express-mysql-session\node_modules\mocha\lib\mocha.js:801:10)
    at Object.exports.singleRun (*snip*\express-mysql-session\node_modules\mocha\lib\cli\run-helpers.js:207:16)
    at exports.runMocha (*snip*\express-mysql-session\node_modules\mocha\lib\cli\run-helpers.js:300:13)
    at Object.exports.handler.argv [as handler] (*snip*\express-mysql-session\node_modules\mocha\lib\cli\run.js:296:3)
    at Object.runCommand (*snip*\express-mysql-session\node_modules\yargs\lib\command.js:238:44)
    at Object.parseArgs [as _parseArgs] (*snip*\express-mysql-session\node_modules\yargs\yargs.js:1072:28)
    at Object.parse (*snip*\express-mysql-session\node_modules\yargs\yargs.js:566:25)
    at Object.exports.main (*snip*\express-mysql-session\node_modules\mocha\lib\cli\cli.js:62:6)
    at Object.<anonymous> (*snip*\express-mysql-session\node_modules\mocha\bin\_mocha:10:23)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:741:12)
    at startup (internal/bootstrap/node.js:285:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:739:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! express-mysql-session@2.1.1 test:unit: `mocha test/unit/ --recursive --reporter spec --ui bdd`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the express-mysql-session@2.1.1 test:unit script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! express-mysql-session@2.1.1 test: `npm run test:unit && npm run test:integration`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the express-mysql-session@2.1.1 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! express-mysql-session@2.1.1 prepublish: `npm run test`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the express-mysql-session@2.1.1 prepublish script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
```

I think windows is being weird or something, I'll investigate it some more later today.

--- a/package.json
+++ b/package.json
@@ -6,8 +6,8 @@
   "scripts": {
     "benchmarks": "./node_modules/.bin/mocha test/benchmarks/ --recursive --reporter spec --ui bdd",
     "lint": "./node_modules/.bin/eslint --config .eslintrc.js *.js test/**/*.js",
-    "prepare": "npm run test # runs before `npm publish`",
-    "pretest": "npm run lint # runs before `npm test`",
+    "prepare": "npm run test",
+    "pretest": "npm run lint",
     "test": "npm run test:unit && npm run test:integration",
     "test:integration": "./node_modules/.bin/mocha test/integration/ --recursive --reporter spec --ui bdd",
     "test:unit": "./node_modules/.bin/mocha test/unit/ --recursive --reporter spec --ui bdd"

